addons/web/static/src/search/control_panel/control_panel.xml

```diff
@@ -42,9 +62,17 @@
                 </div>
 
                 <div class="o_control_panel_navigation d-flex flex-wrap flex-md-nowrap justify-content-end gap-3 gap-lg-1 gap-xl-3 order-1 order-lg-2 flex-grow-1">
+                    <div t-if="(state.topbarInfos.showTopbar || env.isSmall) &amp;&amp; state.topbarInfos.topbarActions?.length" class="d-flex d-md-none">
```

Comment:
FYI, `and` keyword exists in QWeb and so we can use it instead of `&amp;&amp;` 🙂